From 99a8c69a0c9f70d2642dff0c92d76f355a40d233 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Fri, 18 Mar 2016 06:05:16 -0700
Subject: [PATCH 1601/1767] Revert "UBUNTU: SAUCE: fixup: make
 __share_replacedby private to get rid of build warning"

BugLink: http://bugs.launchpad.net/bugs/1379535

This reverts commit 844c52f03735e9d39e76e515e509a103677a71fa.

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 security/apparmor/label.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/label.c b/security/apparmor/label.c
index ee9a573..d80fc50 100644
--- a/security/apparmor/label.c
+++ b/security/apparmor/label.c
@@ -87,7 +87,7 @@ void __aa_update_replacedby(struct aa_label *orig, struct aa_label *new)
 	aa_put_label(tmp);
 }
 
-static void __share_replacedby(struct aa_label *old, struct aa_label *new)
+void __share_replacedby(struct aa_label *old, struct aa_label *new)
 {
 	struct aa_replacedby *r = new->replacedby;
 	new->replacedby = aa_get_replacedby(old->replacedby);
-- 
1.9.3

